avm/res/cache/redis/main.bicep

```diff
@@ -303,6 +304,17 @@ module redis_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.4.1'
   }
 ]
 
+module redis_geoReplication 'linked-servers/main.bicep' = if (!empty(geoReplicationObject)) {
+  name: '${uniqueString(deployment().name, location)}-redis-LinkedServer'
+  params: {
+    redisCacheName: redis.name
+    name: geoReplicationObject.secondaryRedisCacheName
+    linkedRedisCacheResourceId: geoReplicationObject.linkedRedisCacheId
+    linkedRedisCacheLocation: geoReplicationObject.linkedRedisCacheLocation
```

Comment:
Beware, if a parameter should be optional it must be here too. Usually, you can achieve this by adding a question mark like `geoReplocationObject.?linkedRedisCacheLocation`

Reply:
Fixed.